packages/internal/src/__tests__/__snapshots__/graphqlCodeGen.test.ts.snap

```diff
@@ -22,11 +24,11 @@ export type Scalars = {
   Int: number;
   Float: number;
   BigInt: number;
-  Date: string;
-  DateTime: string;
+  Date: Date | string;
```

Comment:
I was curious to see what you decided to do here 🙂  Why did you have to keep the possibility of this being a string?

Reply:
I explained it in a comment, but it got "outdated"


> Important change.

> Redwood has built-in handling for Dates (because we [graphql-scalars.dev/docs/scalars/date-time](https://www.graphql-scalars.dev/docs/scalars/date-time) configured) - but our types were only allowing string. This was a misconfiguration, but not super visible, because we were mapping Prisma types anyway.

> For DateTime fields you can send back Date but also a valid date-string. Note that when you set DateTime in your SDL, even if you format it yourself, it will convert it back to the RFC3339.

> Note also that the code here only affects types we generate, I'm just highlighting what the behaviour actually is and updating the types to match it!